src/pages/home/report/ReportActionsList.js

```diff
@@ -404,8 +405,34 @@ function ReportActionsList({
         if (!markerFound) {
             setCurrentUnreadMarker(null);
         }
+    };
+
+    useEffect(() => {
+        calculateUnreadMarker();
+
+        /* eslint-disable-next-line react-hooks/exhaustive-deps */
```

Comment:
Is disabling of the lint rule necessary here? If so could you add a comment on why as mentioned in the guidelines.
https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#what-is-the-exhaustive-deps-lint-rule-can-i-ignore-it